javacs-lab12/src/com/flatironschool/javacs/WikiSearch.java

```diff
@@ -82,8 +92,11 @@ public WikiSearch and(WikiSearch that) {
 	 * @return New WikiSearch object.
 	 */
 	public WikiSearch minus(WikiSearch that) {
-        // FILL THIS IN!
-		return null;
+    Map<String, Integer> difference = new HashMap<String, Integer>(map);
```

Comment:
Thanks :DDD this is actually indented in the text editor but somehow looks different on Github. Hmmm...